compiler/daml-lf-ast/src/DA/Daml/LF/Ast/World.hs

```diff
@@ -62,16 +62,14 @@ data DalfPackage = DalfPackage
 instance NFData DalfPackage
 
 -- | Construct the 'World' from only the imported packages.
+-- TODO (drsk) : hurraybit: please check that the duplicate package id check here is not needed.
```

Comment:
This check caught a couple of bugs for me in the past. What changed to force us to remove this?

Reply:
We now have several ghc-prims in different directories, but that have the same package id. This fails in here if this check is here.

Reply:
Can we deduplicate at the call site instead of removing this check?